tools/cldr-apps/src/org/unicode/cldr/web/DataSection.java

```diff
@@ -3162,7 +2643,7 @@ public String toJSONString() throws JSONException {
             result.put("hasExamples", hasExamples);
             result.put("xpathPrefix", xpathPrefix);
             result.put("skippedDueToCoverage", getSkippedDueToCoverage());
-            result.put("coverage", getPtype());
+            // result.put("coverage", "comprehensive" /* getPtype() */);
```

Comment:
Assuming this is not needed? I thought this is how the 'coverage' popup menu hides and shows items.

Reply:
For WHAT_GETROW, the client never specified p_covlev, so the DataSection was always constructed with ptype="comprehensive" anyway. The client does the filtering itself depending on what's in the Coverage menu. There is other client-server communication about coverage but it didn't use WHAT_GETROW or json.section.coverage.

Reply:
OK. This is the overall section, not the coverage of an individual line. I think this is a holdover from when the filtering used to be server side. This line is OK then :+1: